PencilCase/PencilCase/scripts/CustomBindings/InitialSelectedTabBinding.ts

```diff
@@ -0,0 +1,26 @@
+﻿class InitialSelectedTabBinding implements KnockoutBindingHandler {
+    public update(element: any, valueAccessor: () => any, allowBindingAccessor: KnockoutAllBindingsAccessor, viewModel: any, bindingContext: KnockoutBindingContext) {
```

Comment:
This custom binding handler makes the tabs control set the default active tab when specific condition is true, and use JQuery selector string to specify which tab will be set as the active one.